interface/main/calendar/modules/PostCalendar/pntemplates/compiled/README

```diff
@@ -1,2 +0,0 @@
-This is a dummy file to ensure this directory is included in software
-package release.
```

Comment:
since this has been fixed (added back interface/main/calendar/modules/PostCalendar/pntemplates/compiled/README), I am going to remove the related comments to keep the review clean.